src/expint.jl

```diff
@@ -0,0 +1,28 @@
+function expint(x::Float64)
+    if x >= 1.0
+        e = exp(-x/2)
+        xinv = inv(x)
+        a = evalpoly(xinv, (1.3319248966682697359e-30, 0.99999999999999993242, 37.702667724670064050, 542.50695317051747114, 3836.6700640600929730, 14346.114698914059563, 28502.272377756659505, 28835.636437354567322, 13473.368739266960527,2387.9894006598233823, 99.579680699439479278))
```

Comment:
simdmath here.

Reply:
I might have to defer to you and @heltonmc here---is there a good way to try and write that in an architecture-agnostic way? It seems like you need to know the lane width (?) at compile time to `pack` them, although I'm probably just missing something.

Reply:
SIMDMath is agnostic to the lane width which was a design goal. Essentially we are relying on LLVM to optimize it for the native architecture. In this case we only have two polynomials so our SIMD vector will be length two (unless we do more fancy things like using a second order horner evaluation which will give us a SIMD width of 4 optimal for avx). Going much further probably won't help here because the length of the polynomial is too short.

Here is an example of how I use SIMDMath.
https://github.com/JuliaMath/Bessels.jl/blob/ab48a96d4c4ea11c5dc70eaf35c1692a58c0fdc1/src/Airy/cairy.jl#L418 

In this case it's just going to be 
```julia
SIMDMath.pack_poly((a, b))
```
But I can work on the expint function more. In this case the autovectorizer is working well though and able to vectorize that function already.